dist/reporttable.js

```diff
@@ -17,229 +17,238 @@
+            return;
+        }
+        this.scroller = $("<div>").addClass(this.options.cssPrefix + "_scrollbar");
+        this.scroller.css({
+            "position":"relative",
+            "overflow-x":"auto",
+            "overflow-y":"hidden",
+            "z-index":6,
+            "width":this.options.scrollContainer.width()
+        });
+        
+        var innerScroll = $("<div>").addClass(this.options.cssPrefix + "_scrollbarInner");
+        innerScroll.css({
+            "width":this.container.width(),
+            "height":0
+        });
+        
+        this.options.scrollContainer.unbind('scroll', this.scrollContainerScrollEvent);
+        this.scrollContainerScrollEvent = $.proxy(this.scrollScrollbar, this);
+        this.options.scrollContainer.scroll(this.scrollContainerScrollEvent);
 
-		innerScroll.html("&nbsp;");
-		this.scroller.scroll($.proxy(this.scrollContainer, this));
-		this.scroller.append(innerScroll);
-		this.options.scrollContainer.after(this.scroller);
-	}
-	
-	ReportTable.prototype.scrollContainer = function(event)
-	{
-		this.options.scrollContainer.scrollLeft(this.scroller.scrollLeft());
-	}
-	
-	ReportTable.prototype.scrollScrollbar = function(event)
-	{
-		this.scroller.scrollLeft(this.options.scrollContainer.scrollLeft());
-	}
-	
-	ReportTable.prototype.getWidthsFromFirstColumn = function()
-	{
-		var firstRow = this.body.find("tr").eq(0);
+        innerScroll.html("&nbsp;");
+        this.scroller.scroll($.proxy(this.scrollContainer, this));
+        this.scroller.append(innerScroll);
+        this.options.scrollContainer.after(this.scroller);
+    }
+    
+    ReportTable.prototype.scrollContainer = function(event)
+    {
+        this.options.scrollContainer.scrollLeft(this.scroller.scrollLeft());
+    }
+    
+    ReportTable.prototype.scrollScrollbar = function(event)
+    {
+        this.scroller.scrollLeft(this.options.scrollContainer.scrollLeft());
+    }
+    
+    ReportTable.prototype.getWidthsFromFirstColumn = function()
+    {
+        var firstRow = this.body.find("tr").eq(0);
 
-		var widths = [];
-		firstRow.find("td").each(function(index) {
-			widths.push($(this).width());
-		});
-		return widths;
-	}
-	
-	
-	ReportTable.prototype.defaultOptions = {
-		offsetHeaderHeight:0,
-		offsetFooterHeight:0,
-		cssPrefix:"reporttable",
-		pollInterval:400,
-		resizePollInterval:2000,
-		headerCell:"th",
-		scrollContainer:null //a jquery object
-	}
+        var widths = [];
+        firstRow.find("td").each(function(index) {
+            widths.push($(this).width());
+        });
+        return widths;
+    }
+    
+    
+    ReportTable.prototype.defaultOptions = {
+        offsetHeaderHeight:0,
+        offsetFooterHeight:0,
+        cssPrefix:"reporttable",
+        pollInterval:400,
+        resizePollInterval:2000,
+        headerCell:"th",
+        scrollContainer:null,
+        scrollDefaultObject:null
```

Comment:
Should we name this option `scrollEventObject`, or something, so the purpose is clearer from the name?